tests/acceptance/features/webUICreateFilesFolders/createFolders.feature

```diff
@@ -67,3 +67,28 @@ Feature: create folders
     Then folder "sub-folder" should be listed on the webUI
     When the user reloads the current page of the webUI
     Then folder "sub-folder" should be listed on the webUI
+
+
+  Scenario: Create folders with dot in the name
+    When the user creates the following folders using the webUI
+      | folders  |
+      | fo.      |
+      | fo.1     |
+      | fo...1.. |
+      | ...      |
+      | ..fo     |
+      | fo.xyz   |
+    Then the following folders should be listed on the webUI
+      | folders  |
+      | fo.      |
+      | fo.1     |
+      | fo...1.. |
+      | ...      |
+      | ..fo     |
+      | fo.xyz   |
+
+
+  @skipOnOC10 @issue-5518
```

Comment:
Note: On oC10, `.part` files are not allowed. So this scenario will always be tagged `skipOnOC10`

The issue is about improving the current behavior of `web` with oC10. It should give the user some good feedback about why the resource cannot be created. When that is improved, then an extra scenario can be added here that demonstrates the behavior with oC10 (and will be skipped on OCIS)